path_roles.go

```diff
@@ -87,6 +89,14 @@ func pathsRole(b *azureSecretBackend) []*framework.Path {
 					Type:        framework.TypeString,
 					Description: "JSON list of Azure groups to add the service principal to.",
 				},
+				"sign_in_audience": {
+					Type:        framework.TypeString,
+					Description: "Specifies the security principal types that are allowed to sign in to the application.",
+				},
+				"tags": {
+					Type:        framework.TypeCommaStringSlice,
+					Description: "Azure tags to attach to an application.",
+				},
```

Comment:
It seems like non-KV formats work, but to be sure we are fine if tags do not exactly follow a `"key:value"` format?
```
vault write azure/roles/my-role ... tags="just-a-string"
vault write azure/roles/my-role ... tags="not-a-colon=delimiter"
etc.
```
Are working just fine. If this is on purpose it should probably be included in tests so clearly signal to future maintainers that this is on purpose and not a missing validation.

Reply:
This is intentional, for app registrations Azure only allows tags to be passed as an array of strings unfortunately. 
https://feedback.azure.com/d365community/idea/ec2330ed-b125-ec11-b6e6-000d3a4f0789